src/displayapp/screens/Music.cpp

```diff
@@ -223,7 +243,11 @@ void Music::UpdateLength() {
 
 void Music::OnObjectEvent(lv_obj_t* obj, lv_event_t event) {
   if (event == LV_EVENT_CLICKED) {
-    if (obj == btnVolDown) {
+    if (obj == btnForward) {
```

Comment:
This looks like a switch:
```cpp
switch (obj) {
  case btnForward:
    ...
}

Reply:
I can not make this a switch since the obj variable type is an object and not an integral type. There is no way for the compile to convert the obj to an integer needed for the switch statement.